### PR TITLE
Remove classes From Testing Exercise

### PR DESCRIPTION
**Pull request by CheezItMan**
Update to the JS testing exercise, the textbook curriculum PR is coming shortly.  In this PR I:

- Removed classes from the exercise, so it's more flexible as to when we put it in.
- Split Pangram and Palindrome into two file, the latter is optional for those who finish quickly.

Questions:

Is this too much for them to do, I like it since it gets students to practice writing JS functions while it's still new, but we could introduce testing with a shorter intro.  I could change this to have them write tests to identify a bug in a version of isPangram.  That would be faster certainly, but they wouldn't practice writing as much JS.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "anagram",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (kaidamasaki) [inline package.json:3]: This seems like it might be a breaking change so we should probably bump the major version.  😉

**Review by kaidamasaki — APPROVED** [review]

**Review by kaidamasaki — APPROVED** [review]

File: src/is_pangram.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const isPangram = function isPangram(text) {
+
+};
```
Comment (droberts-sea) [inline src/is_pangram.js:3]: Here you include the name of the function twice, but you do not do this for `isPalindrome`.

File: spec/is_pangram.spec.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    // Act
+
+    // Assert
+
+  });
+
+  test('empty sentence', () => {});
+
+  test('pangram with underscores instead of spaces works', function textWithUnderscores() {
+    // Arrange
```
Comment (droberts-sea) [inline spec/is_pangram.spec.js:39]: Why is `empty sentence` not broken out into A/A/A?

File: README.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ insensitive. Input will not contain non-ASCII symbols.
 
 ## Problem Description #2 Palindrome
 
-For the next method write a method which will determine if the `TextBlock` is a palindrome.  A Palindrome is a word or phrase which is spelled the same backwards as forwards.
+For the next function write a method which will determine if the input string is a palindrome.  A Palindrome is a word or phrase which is spelled the same backwards as forwards.
```
Comment (droberts-sea) [inline README.md:36]:
```suggestion
Write a function which will determine if the input string is a palindrome.  A Palindrome is a word or phrase which is spelled the same backwards as forwards.
```

**Review by droberts-sea — CHANGES_REQUESTED** [review]

**Pull request merged**